doc/source/contribute/documentarian.rst

```diff
@@ -0,0 +1,141 @@
+Contributing as a documentarian
+###############################
+
+.. grid:: 1 1 3 3
+
+    .. grid-item-card:: :fa:`pencil` Write documentation
+        :padding: 2 2 2 2
+        :link: write-documentation
+        :link-type: ref
+
+        Explain how to get started, use, and contribute to the project.
+
+    .. grid-item-card:: :fa:`laptop-code` Add a new example
+        :padding: 2 2 2 2
+        :link: write-examples
+        :link-type: ref
+
+        Showcase the capabilities of PyWorkbench by adding a new example.
+
+    .. grid-item-card:: :fa:`file-code` Build the documentation
+        :padding: 2 2 2 2
+        :link: build-documentation
+        :link-type: ref
+
+        Render the documentation to see your changes reflected.
+
+.. _write-documentation:
+
+Write documentation
+===================
+
+The documentation generator used in PyWorkbench is `Sphinx`_. Most of the documents
+are written in `reStructuredText`_. Some parts of the documentation, like the
+:ref:`examples <Examples>`, use mix of `markdown`_ and Python. If
+you are interested in writing examples, see the :ref:`writing examples <write-examples>`
+section.
+
+The documentation is located in the ``doc/source`` directory. The landing page
+is declared in the ``doc/source/index.rst`` file. The rest of the files contain
+the main pages of different sections of the documentation. Finally, the
+``doc/source/_static/`` folder contains various assets like images, and CSS
+files.
+
+The layout of the ``doc/source`` directory is reflected in the slug of the
+online documentation. For example, the
+``doc/source/contribute/documentarian.rst`` renders as
+``https://workbench.docs.pyansys.com/contribute/documentarian``.
+
+Thus, if you create a new file, it important to follow these rules:
+
+- Use lowercase letters for file and directory names
+- Use short and descriptive names
+- Use hyphens to separate words
+- Play smart with the hierarchy of the files and directories
+
+All files need to be included in a table of contents. No dangling files are
+permitted. If a file is not included in the table of contents, Sphinx raises a
+warning that makes the build to fail.
```

Comment:
```suggestion
You must include all files in a table of contents. No orphan files are
permitted. If a file is not included in the table of contents, Sphinx raises a
warning that causes the build to fail.
```
I know we want to steer away from using ``index.rst`` files for other than the main landing page. I'm just wondering if this needs to be stated, especially given that so many libraries use this name, making the "table of contents" reference used here unfamiliar?